### PR TITLE
Add libray-occupancy extension

### PR DESCRIPTION
**Pull request by Alleluia23**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/libray-occupancy/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "libray-occupancy",
+  "title": "How Busy Is the Library Now",
```
Comment (pernielsentikaer) [inline extensions/libray-occupancy/package.json:4]:
```suggestion
  "title": "Library in University of St Andrews",
```
Can we come up with better naming of the extension?

File: extensions/libray-occupancy/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Hello World Changelog
```
Comment (pernielsentikaer) [inline extensions/libray-occupancy/CHANGELOG.md:1]: change the hello world here too

**Comment by erics118** [comment]
BTW the extension folder and package.json `name` field is `libray-occupancy`, but it should be `library-occupancy`. There's a missing `r`.

Also, maybe the extension should be named `st-andrews-library-occupancy`, as this is specific to only the St. Andrews University

File: extensions/libray-occupancy/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "title": "Main Library in University of St Andrews",
+  "description": "How busy is the main library now?",
+  "icon": "command-icon.png",
+  "author": "Harry",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Library Occupancy",
+      "description": "Template with a plain detail view",
```
Comment (erics118) [inline extensions/libray-occupancy/package.json:13]: This probably was missed when copying over the template

Comment (pernielsentikaer) [reply extensions/libray-occupancy/package.json:13]: That would indeed make sense to change

**Comment by pernielsentikaer** [comment]
Hi @Alleluia23 👋 

Could you check the comments in this thread by @erics118

**Comment by pernielsentikaer** [comment]
Closing since it's merged in another PR

**Pull request closed without merging**